_includes/components/who-uses-login.html

```diff
@@ -0,0 +1,15 @@
+  <div class="bg-primary-lightest">
+    <div class="container who-uses-login">
+      <div class="partners list">
+        {{ include.col1 | markdownify }}
+      </div>
+    </div>
+  </div>
+  
+  <div>
+    <div class="container who-uses-login">
+      <div class="security">
+        {{ include.col2 | markdownify }}
+      </div>
+    </div>
+  </div>
```

Comment:
Minor: Can we de-indent this, since it's not nested in anything?

```suggestion
<div class="bg-primary-lightest">
  <div class="container who-uses-login">
    <div class="partners list">
      {{ include.col1 | markdownify }}
    </div>
  </div>
</div>

<div>
  <div class="container who-uses-login">
    <div class="security">
      {{ include.col2 | markdownify }}
    </div>
  </div>
</div>
```